pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -68,6 +72,14 @@ var (
 	tracer      = otel.Tracer("github.com/grafana/grafana/pkg/services/dashboards/service")
 )
 
+const (
+	k8sDashboardCleanupInterval          = 30 * time.Second
+	k8sDashboardCleanupTimeout           = 22 * time.Second
+	k8sDashboardCleanupBatchSize         = 10
```

Comment:
Small nit, but could we make this somehow configurable?

Reply:
Ok